### PR TITLE
Displays missing item in battle

### PR DESCRIPTION
**Pull request by mario51y1**
refers #41

### CODE REVIEW

**Comment by tagniam** [comment]
@mario51y1 It looks like there's a bug where if you repeatedly use an item not in the inventory, the enemy will heal themselves instead of doing nothing.

File: Turn/src/Game.cpp [hunk 1]
```diff
@@ -274,8 +274,10 @@ void Game::Battle(){
         _Player->DisplayHUD(_Enemy);
         _Enemy->DisplayHUD();
 
+		int damagePlayer = _Player->Attack();
         // Player's turn to attack Enemy.
-        _Enemy->TakeDamage(_Player->Attack());
+
+		_Enemy->TakeDamage(damagePlayer);
```
Comment (tagniam) [inline Turn/src/Game.cpp]: Bug occurs here, when `damagePlayer = 2`, 2 points are added to enemy's health (which makes it look like it's healing)

**Comment by mario51y1** [comment]
Yeah, thought I made the if statement that checks the damage but seems it's missing.

**Comment by tagniam** [comment]
Could you make -2 a constant, like SKIP_TURN? This will ensure readability

File: Turn/src/Player.cpp [hunk 2]
```diff
@@ -114,8 +114,10 @@ int Player::Attack(){
 			{
 				PlaySecondaryAttack();
 				return BowAndArrow();
-			}
```
Comment (tagniam) [inline Turn/src/Player.cpp]: Missing ending curly brace

File: Turn/src/Player.cpp [hunk 3]
```diff
@@ -126,24 +128,30 @@ int Player::Attack(){
...
-			break;
 		case 7:
 			// Player sharpens their weapon with a whetstone.
 			// Does not execute if there are no whetstones in inventory.
 			// No damage is done to the enemy.
 			if (whetstones > 0) {
 				UseWhetstone();
 				return 0;
+			} else {
+				cout << "No whetsones in the inventory!" << endl;
```
Comment (tagniam) [inline Turn/src/Player.cpp]: Spelling: whetstones

File: Turn/src/Player.cpp [hunk 4]
```diff
@@ -114,8 +114,10 @@ int Player::Attack(){
 			{
 				PlaySecondaryAttack();
 				return BowAndArrow();
-			}
-			break;
+			else
+				cout << "No arrows in the inventory!" << endl;
```
Comment (tagniam) [inline Turn/src/Player.cpp]: Wrap this else block with curly braces

File: Turn/src/Player.cpp [hunk 5]
```diff
@@ -126,24 +128,30 @@ int Player::Attack(){
 			// Does not execute if there are no bombs in the inventory.
 			if (bombs > 0)
 				return UseBomb();
-			break;
+			else
```
Comment (tagniam) [inline Turn/src/Player.cpp]: Wrap this else block with curly braces

**Comment by tagniam** [comment]
Also, weirdly when I try to use a non-existent item, the message is displayed twice as long.

**Comment by mario51y1** [comment]
Ok, so is something else missing? Had some troubles with visual sutudio and I couldnt test if everything is working as suposed.

**Comment by tagniam** [comment]
Thanks for the patience while I reviewed this PR @mario51y1. Everything's good now, I just had to fix a little bug in the code but it's all working now. Happy hacktoberfest! 🎃

**Comment by mario51y1** [comment]
Thanks for your patience as well working wiht a newbie in GitHub, I'm happy that evertyhing is working now. Happy Hacktoberfest too! 🎃

**Comment by tagniam** [comment]
I was in your position as a newbie on github just a few years ago, everyone starts somewhere :smile: Hope you continue to contribute to open source in the future!

**Pull request merged**